src/test/java/org/jabref/logic/importer/fetcher/CompositeSearchBasedFetcherTest.java

```diff
@@ -0,0 +1,110 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Stream;
+
+import org.jabref.logic.bibtex.FieldContentFormatterPreferences;
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.ImportFormatPreferences;
+import org.jabref.logic.importer.SearchBasedFetcher;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.testutils.category.FetcherTest;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@FetcherTest
+public class CompositeSearchBasedFetcherTest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CompositeSearchBasedFetcherTest.class);
+
+    @Test
+    public void performSearchWithoutFetchers() {
+        Set<SearchBasedFetcher> empty = new HashSet<>();
+        CompositeSearchBasedFetcher fetcher = new CompositeSearchBasedFetcher(empty);
+
+        List<BibEntry> result = fetcher.performSearch("quantum");
+
+        Assertions.assertTrue(result.isEmpty());
```

Comment:
please don't use assertTrue(empty) but assertEquals(..., emptyList) since this gives a better error message